### PR TITLE
Storage API docs

### PR DESCRIPTION
**Pull request by mosic**

### CODE REVIEW

File: packages/sdk/README.md [hunk 1]
```diff
@@ -300,6 +314,20 @@ const key = 'my-key'
 const data = await sdk.storage.get(key)
 ```
 
+### storage.delete()
+
+Deletes a file from the app's manifest.
```
Comment (PaulLeCam) [inline packages/sdk/README.md]: Could you use another term than "manifest" here please? It's an implementation detail, I don't think it should leak to the doc.

**Comment by PaulLeCam** [comment]
@mosic could you also edit the website docs please? Here is the file: https://github.com/MainframeHQ/mainframe-os/blob/master/docs/sdk.md

File: packages/sdk/README.md (hunk 1 above)
Comment (mosic) [reply packages/sdk/README.md]: I actually paused when I was writing that, but wasn't sure what would be better.

Maybe "Deletes an entry from the list of files available to the app"?

**Comment by mosic** [comment]
@PaulLeCam Sure :+1:

**Pull request merged**